ocs_ci/deployment/deployment.py

```diff
@@ -662,7 +662,6 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
         self.do_deploy_submariner()
         self.do_gitops_deploy()
         self.do_deploy_ocs()
-        self.do_deploy_oadp()
```

Comment:
Please check this comment https://bugzilla.redhat.com/show_bug.cgi?id=2280657#c13 and do the needful based on the answer from Talur

Reply:
Based on the latest comment https://bugzilla.redhat.com/show_bug.cgi?id=2280657#c14, fix has been reverted and need to install OADP manually. Closing this PR.